internal/injector/testdata/injector/chi5-newroute-dotimport/config.yml

```diff
@@ -16,6 +16,7 @@ aspects:
             }()
 
 syntheticReferences:
+  github.com/go-chi/chi/v5: true
```

Comment:
That reference exists in the original file, so it should be be reported as a synthetic one? It's not a big deal here (it'd be filtered later one due to it already being present in the `importcfg` file, but still...)

Reply:
This is part of the idea to mitigate shadowing when dot imports are used